hazelcast/src/test/java/com/hazelcast/map/QueryListenerTest.java

```diff
@@ -46,11 +46,11 @@
 
     @Test
     public void testMapQueryListener() throws InterruptedException {
+        Config config = getConfig();
         TestHazelcastInstanceFactory nodeFactory = createHazelcastInstanceFactory(3);
-        Config cfg = new Config();
-        HazelcastInstance instance1 = nodeFactory.newHazelcastInstance(cfg);
-        HazelcastInstance instance2 = nodeFactory.newHazelcastInstance(cfg);
-        HazelcastInstance instance3 = nodeFactory.newHazelcastInstance(cfg);
+        HazelcastInstance instance1 = nodeFactory.newHazelcastInstance(config);
```

Comment:
I would just stick these instances in an array. Because a lot of noise is created here.